scripts/mkcloud

```diff
@@ -173,6 +173,9 @@ host_mtu=$(determine_mtu)
 iscloudver 7plus && [[ $arch == x86_64 ]] && \
     [[ -n $want_efi ]] && firmware_type="uefi"
 
+: ${bmc_user:=root}
+: ${bmc_password:=cr0wBar!}
```

Comment:
should be moved into mkcloud-common so that it is also defined in mkphyscloud-like cases that run without mkcloud

Reply:
@bmwiedemann done